app/src/main/java/com/example/tanya/godeltechchallengeandroid/ui/start/StartActivity.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.tanya.godeltechchallengeandroid.ui.start
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import com.example.tanya.godeltechchallengeandroid.R
+import com.example.tanya.godeltechchallengeandroid.ui.next.HomeActivity
+import dagger.android.support.DaggerAppCompatActivity
+import kotlinx.android.synthetic.main.activity_start.*
+import javax.inject.Inject
+
+class StartActivity : DaggerAppCompatActivity(), StartContract.View {
+
+    @Inject lateinit var presenter: StartContract.Presenter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_start)
+
+        presenter.bindView(this)
+
+        presenter.loadData()
```

Comment:
Why do you need the extra presenter call (`loadData()`)? What is wrong if you start execute your business logic as soon as the `bindView(this)` function is called?